feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch

```diff
@@ -0,0 +1,26 @@
+--- a/src/ap/ieee802_11.c
++++ b/src/ap/ieee802_11.c
+@@ -4563,11 +4563,6 @@ static void handle_assoc(struct hostapd_
+ 	}
+ #endif /* CONFIG_FILS */
+ 
+-	if (resp >= 0)
+-		reply_res = send_assoc_resp(hapd, sta, mgmt->sa, resp, reassoc,
+-					    pos, left, rssi, omit_rsnxe);
+-	os_free(tmp);
+-
+ 	ubus_resp = hostapd_ubus_handle_event(hapd, &req);
+ 	if (ubus_resp) {
+ 		wpa_printf(MSG_DEBUG, "Station " MACSTR " assoc rejected by ubus handler.\n",
+@@ -4576,6 +4571,11 @@ static void handle_assoc(struct hostapd_
+ 		goto fail;
+ 	}
```

Comment:
Can we move this even up? I mean why do the other stuff if we are not going to associate the client in case of low rssi.